contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -397,77 +511,50 @@ Already have a warehouse? You can [link it with ours](/docs/data-warehouse) to a
-    classes="rounded"
-    alt="product analytics pricing"
-/>
+</CalloutBox>
 
-PostHog charges for usage – i.e. analytics events, recorded replays, and so on. **Amplitude** charges based on monthly tracked users (MTUs).
+## Price comparison
 
-In the comparison, we're assuming 75 identified events per monthly tracked user (MTU), which is the typical usage for most PostHog users, though many customers send less than this.
+### PostHog pricing philosophy
 
-| **Monthly tracked users** | **Amplitude Growth** | **PostHog** | &nbsp                                                        |
-| ------------------------- | -------------------- | ----------- | ------------------------------------------------------------ |
-| 1k                        | $61                  | $0          | <strong class="text-green">$732 per year cheaper</strong>    |
-| 2.5k                      | $99                  | $0          | <strong class="text-green">$1,888 per year cheaper</strong>  |
-| 5k                        | $124                 | $0          | <strong class="text-green">$1,488 per year cheaper</strong>  |
-| 10k                       | $186                 | $0          | <strong class="text-green">$2,232 per year cheaper</strong>  |
-| 25k                       | $249                 | $217        | <strong class="text-green">$1,128 per year cheaper</strong>  |
-| 50k                       | $561                 | $430        | <strong class="text-green">$1,572 per year cheaper</strong>  |
-| 75k                       | $841                 | $625        | <strong class="text-green">$2,592 per year cheaper</strong>  |
-| 100k                      | $1,061               | $820        | <strong class="text-green">$2,892 per year cheaper</strong>  |
-| 150k                      | $1,584               | $1,207      | <strong class="text-green">$4,524 per year cheaper</strong>  |
-| 200k                      | $2,100               | $1,600      | <strong class="text-green">$6,000 per year cheaper</strong>  |
-| 300k                      | $3,150               | $2,091      | <strong class="text-green">$12,708 per year cheaper</strong> |
+**PostHog charges based on usage**. Each product (analytics, session replay, feature flags, etc.) comes with a [generous free tier](/pricing), and once you exceed those limits you pay only for what you use. Pricing is fully transparent and published publicly, with per-unit rates you can calculate in advance. You can also set billing caps per product to avoid unexpected costs.
 
-We also provide numerous ways to [control how many events you send](/tutorials/fewer-unwanted-events) and reduce your costs, including the option to send cheaper **anonymous events**.
+This usage-based approach means costs scale with your actual activity, whether that's events, surveys, or recordings, rather than the number of users in your product.
```

Comment:
```suggestion
This usage-based approach means costs scale with your actual activity, whether that's events, surveys, or recordings, rather than the number of users in your product. It also means you can easily reduce your bill by changing how you track users to send fewer events, or record fewer sessions.
```